src/components/Checkbox.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const configStyle = {
```

Comment:
Move this object in constants file and rename the properties names. true and false are reserved keys in javascript and it's not ok to use them this way.